packages/integrations/useSortable/index.ts

```diff
@@ -51,12 +51,15 @@ export function useSortable<T>(
 
   const start = () => {
     const target = (typeof el === 'string' ? document?.querySelector(el) : unrefElement(el))
-    if (!target)
+    if (!target || sortable !== undefined)
```

Comment:
if (!target || sortable)

Reply:
I think this is a code style problem 😂, the type of sortable is ```Sortable | undefined```, if sortable doesn't equal undefined, it means sortable exists.